### ISSUE TITLE
Error when restoring a file with targetcli on Suse OS

### ISSUE THREAD

**Issue by juadk** [saltstack-formulas/iscsi-formula#15]
In Suse OS, the comments include in /etc/target/saveconfig.json by man5.jinja causes an issue when you want to restore it.

Example:
![restoreconfig](https://user-images.githubusercontent.com/6025636/52846530-1d266400-310a-11e9-86e6-dbf757261187.png)

**Comment by noelmcloughlin** [comment]
This is a formula bug - comments are not allowed in JSON.  Maybe you could raise PR to fix.

**Comment by juadk** [comment]
Sorry I forgot to add the PR number here after submitted it.

**Comment by noelmcloughlin** [comment]
Thanks.  Another PR would be really useful - no comments allowed in JSON - I merged your PR to unblock you on Suse.
